<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="log" default="true" />
    <arg name="algo" default="bug2" />
    <arg name="initial_x" default="0" />
    <arg name="initial_y" default="8" />
    <arg name="initial_z" default="0" />
    <arg name="des_x" default="4" />
    <arg name="des_y" default="8" />
    <param name="log" value="$(arg log)" />
    <param name="algo" value="$(arg algo)" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="initial_z" value="$(arg initial_z)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="com760_group19" type="follow_wall.py" name="wall_follower" />
    <node pkg="com760_group19" type="go_to_point.py" name="go_to_point" />
    <node pkg="com760_group19" type="bugs.py" name="bugs" output="screen" />
</launch>